doc/conf.py

```diff
@@ -225,69 +223,15 @@
 htmlhelp_basename = "PennyLanedoc"
 
 # -- Xanadu theme ---------------------------------------------------------
-html_theme = "xanadu"
+html_theme = "pennylane"
 
 # Xanadu theme options (see theme.conf for more information).
 html_theme_options = {
-    "navbar_logo_path": "_static/logo.png",
-    "navbar_wordmark_path": "_static/pennylane.svg",
-    # Specifying #19b37b is more correct but does not match the other PL websites.
-    "navbar_logo_colour": "#2d7c7f",
-    "navbar_home_link": "https://pennylane.ai",
-    "navbar_left_links": [
```

Comment:
These seem to be removed for good (the new theme doesn't define them explicitly). That's something we'd like to do, right?

Reply:
The new theme does define them, in the `pennylane_sphinx_theme/__init__.py` :)